### ISSUE TITLE
Help getting started

### ISSUE THREAD

**Issue by apacha** [wmde/wikibase-release-pipeline#401]
Hi,

thank you very much for putting together this docker-compose setup. I was very happy when I ran the example and quickly saw a running MediaWiki instance where I could create pages, items and properties. However, I got stuck and after reading various tutorials and pages for several hours, I decided to post a question here, as it might help others too. 

We are interested in hosting our own WikiData instance for collecting information about composers and musical works. Basically we only want to use the WikiData web-interface and the API behind it (as far as I understand WikiData is an instance of Wikibase, which is an extension to MediaWiki; so we want to run our own MediaWiki/Wikibase).

After starting the instances, I've created a few items and properties, but when I try to link them it doesn't seem to work. In general it feels like there are several things that don't work out-of-the-box (which is fine, as long as I can find the information somewhere).

![Screenshot 2023-01-23 at 11 49 13](https://user-images.githubusercontent.com/594497/214031333-30b3ae74-5c95-42aa-b4d8-2fd0fec4c1b6.png)
![Screenshot 2023-01-23 at 11 48 51](https://user-images.githubusercontent.com/594497/214031339-8378bdd9-7ae9-431c-b653-803af1a73bd7.png)

My main questions are:
* Where can I find setup/configuration instruction (e.g., when running everything on my localhost, how to connect the various services with each other)? Are they all connected automatically? Do I need to change something in the `.env` for it to work correctly? How can I pass configuration variables from the `.env` to the LocalSettings.php inside the docker container? When I have a look at the `LocalSettings.php` it just says `TODO pass in the rest of this with env vars?`.
* What steps are necessary to enable basic functionality (e.g., adding a statement to an item)?
* What steps are necessary to linking an item to a page? I found https://www.mediawiki.org/wiki/Wikibase/Installation#Enable_Sitelinks but it is missing crucial information and completely lacks information on how to do any of these steps with the docker-compose installation (I tried to follow the steps, but it didn't change anything).
![Screenshot 2023-01-23 at 13 35 03](https://user-images.githubusercontent.com/594497/214041470-f424ef19-da7d-4bdb-bffc-00c8b5fad455.png)
![Screenshot 2023-01-23 at 12 35 13](https://user-images.githubusercontent.com/594497/214041474-62bfbb08-f11e-492e-826a-0f07a1089e67.png)
* Are there some good step-by-step resources that could help me setting up Wikibase to do a thorough investigation whether we could use it for our use-case or not?

**Comment by valeriewmde** [comment]
Hey @apacha,

we read through your request and thought it'd be an idea for you to set up a Wikibase Cloud! 
There, the installation process is easier to go through, and the hosting does not need to come from you either. 
If you're interested, have a look at [](https://wikiba.se/ )or [](https://www.wikibase.cloud/) to learn more.

If that's not what you're currently looking for, please give us a sign, so we can continue to support you to answer your questions on how to self-host. 

Cheers!

Valerie

**Comment by apacha** [comment]
Dear @valeriewmde,

thanks for your response. We happily received the invitation last Friday to the Wikibase Cloud beta and are evaluating it right now. Most basic features now work correctly in that setup (like adding properties to items). From our side we fully support this requirement, stated in this document: https://wbstakeholder.group/projects/institutional-requirements#automatic-maintenance-processes-and-updating-cascades-should-work-out-of-the-box about deployment (as well as the other requirements) and are happy to help where we can.

We will do our evaluation now with what is possible on Wikibase Cloud and then I'll post again here, if further assistance for setting up a self-hosted instance is required.

Thanks and best regards
Alex

**Comment by dimasjackson** [comment]
Hi!

I'm having the same dificulties of @apacha to add statements in my local wikibase instance. Also, in page Special:Version the version of elasticsearch appears blank. I tried wikibase cloud and everything worked, but I would like to set up wikibase locally. Please, could you help me?

Thanks.

**Comment by apacha** [comment]
Given that this ticket is still open, I would like to post an update and close it. We've evaluated Wikibase and especially Wikibase Cloud seems to be a great option for people that want a simple Wikibase installation. However, if you want to set it up yourself, you will need assistance, as the documentation doesn't cover all the details that it takes to actually bring up a fully operational Wikibase instance and it's related services. We've also been in contact with different Wikibase consultants that confirmed this issue, but their costs were too high for us to move forward with Wikibase, so we've opted to build our own solution (with Django) and we're very happy with that decision. 

I'd love to use Wikibase in the future, once this technical entrance barrier is gone. And as stated above: If you want Wikibase-as-a-service, I believe Wikibase Cloud is a great option for you (even if there are quite a few things that require special knowledge, e.g., sorting the properties through this magic page: https://enote-playground.wikibase.cloud/wiki/MediaWiki:Wikibase-SortedProperties).
